src/connectors/kujira/kujira.config.ts

```diff
@@ -0,0 +1,131 @@
+import { BigNumber } from 'bignumber.js';
+import { ConfigManagerV2 } from '../../services/config-manager-v2';
+import { getNotNullOrThrowError } from './kujira.helpers';
+
+const configManager = ConfigManagerV2.getInstance();
+
+export interface NetworkConfig {
+  name: string;
+  nodeURL: string | null;
+  chainId: string;
+}
+
+export namespace KujiraConfig {
+  export const config = {
+    chainType: 'KUJIRA',
+    tradingTypes: ['CLOB_SPOT'],
+    chain: 'kujira',
+    networks: new Map<string, NetworkConfig>(
+      Object.entries(configManager.get(`kujira.networks`))
+    ),
+    availableNetworks: [
+      {
+        chain: 'kujira',
+        networks: Object.keys(configManager.get(`kujira.networks`)),
+      },
+    ],
+    connector: 'kujira',
+    prefix: configManager.get('kujira.prefix') || 'kujira',
+    accountNumber: configManager.get('kujira.accountNumber') || 0,
+    nativeToken: 'KUJI',
+    gasPrice: BigNumber(configManager.get('kujira.gasPrice') || 0.00125),
+    gasPriceSuffix: 'ukuji',
+    gasLimitEstimate: BigNumber(
+      configManager.get('kujira.gasLimitEstimate') || 0.009147
+    ),
+    tokens: {
+      url:
+        configManager.get(`kujira.tokens.url`) ||
```

Comment:
Kujira tokens should be fetched just like all other chains, i.e. each network in `conf/kujira.yml` defines `tokenListSource` and `tokenListUrl` (FILE) and the token list is saved in `conf/lists`.

Reply:
This is something that has changed while we were developing so it is working differently indeed.
But although HB now has a standard for the tokenList there's no standard for the marketList, which is very needed too.
We can change the names to tokenListSource, tokenListUrl and save it there on conf/lists, but we still need a place to save the markets list as well.

In any case, those things related to the `kujira.yml` if they are not wrong or causing problems, I'd like to propose to leave them this way, as you know each connector might have very different configuration files and for the tokenListType, from the value informed, we already know if we should fetch it from an url or from a file, so it's not really needed. As for now HB doesn't have a section for the markets information to be saved.

Would that work for you @fengtality? I think the Kujira users, as long as our yml has all the explanations, will not have problems setting it up.